src/EKI.jl

```diff
@@ -134,6 +147,43 @@ function update_ensemble!(eki::EKIObj{FT}, g) where {FT}
 
     compute_error(eki)
 
+    # Check convergence
+    cov_new = cov(eki.u[end], dims=1)
+    cov_threshold = 0.01
```

Comment:
Should we make `cov_threshold` an input argument to `update_ensemble!` (with a default value)? 
At the moment I can see two different ways users will run an EKI:

1. Directly apply `update_ensemble!` without doing a step size search first. In this case the user will get a warning if the ensemble collapses faster than the rate implied by the hard-coded `cov_threshold`.
2. Run `find_eki_step` to find a good step size before the call to `update_ensemble!`. But with the current implementation it's possible that the user is only concerned about collapse speeds greater than what triggers the warning in `update_ensemble`, so he would get the warning despite having chosen a step size that's consistent with the maximum collapse speed he is willing to accept. Conversely, a user might want the ensemble to collapse more slowly and would like to see a warning already at a smaller decrease of the ensemble covariance than what's currently implemented.

Reply:
You are right, I can see how it would be annoying to get the warning even when the user wants fast convergence (e.g. just for calibrating). I will modify this by adding `con_threshold` as a kwarg set to 0.01 by default.